### ISSUE TITLE
[healthmgr] heron-aurora-scheduler.jar is needed in healthmgr

### ISSUE THREAD

**Issue by huijunw** [twitter/heron#2873]
The present heron-core.tar.gz does not include heron-aurora-scheduler.jar file. This jar file is needed in https://github.com/apache/incubator-heron/blob/master/heron/healthmgr/src/java/org/apache/heron/healthmgr/HealthManager.java#L394

![auroraerror](https://user-images.githubusercontent.com/23125934/39208454-e133ea3a-47b7-11e8-88e3-8c1aeb666ee1.PNG)

**Comment by ashvina** [comment]
`heron-core` does not contains `yarn` or `aurora` scheduler (https://github.com/apache/incubator-heron/blob/master/tools/rules/heron_core.bzl#L37). To me it seems it should not contain any other schedulers too. The executor does not need any scheduler jars, since the scheduler starts before executors and the scheduler itself launches all the executors. When scheduler is running as a service it is contacted using scheduler independent http calls. In the other non-service cases, the scheduler is embedded in the client (https://github.com/apache/incubator-heron/blob/master/tools/rules/heron_client.bzl#L32). Perhaps this needs to be asked in a new issue. 
@maosongfu, @kramasamy, @neng WDYT?

w.r.t. this issue, the solution would be add the non-service scheduler (aurora in this case) to the `healthmgr` jar. However I am not sure if inclusion of aurora jar is sufficient. Any files needed by aurora scheduler need to be present on `container-0`. Is that feasible?
@huijunw Are you able to run the `healthmgr` in `container-0`

**Comment by kramasamy** [comment]
I agree - introducing scheduler dependency into heron core - is not a good idea.

**Comment by huijunw** [comment]
as long as `heron-aurora-scheduler.jar` is available in class path for healthmgr, it is fine to me.

**Comment by kramasamy** [comment]
@huijunw - can you explain why Aurora scheduler jar is needed in the core - not clear why?
